PRIVACY.md

```
@@ -0,0 +1,114 @@
+* Location data
+* Camera or microphone access
+* Authentication or user accounts
+* Cloud databases or server-side storage
+* Third-party APIs
+* Wearable-device connectivity
+* Analytics
+* Data synchronization
+* Data export
+* Specific data-retention periods
+
+If none of these apply, this section can be removed.]
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)